src/plopp/backends/matplotlib/line.py

```diff
@@ -159,8 +159,11 @@ def _make_line(
                 fmt="none",
             )
 
-        if self.label:
-            self._ax.legend()
+        if self.label and self._canvas._legend:
+            leg_args = {}
+            if isinstance(self._canvas._legend, (list, tuple)):
+                leg_args = {'loc': self._canvas._legend}
+            self._ax.legend(**leg_args)
```

Comment:
Should we throw a `TypeError` if the `legend` argument is not `bool` or `tuple`...?
Or is it not really worth adding...?